src/App.js

```diff
@@ -1,13 +1,65 @@
 import React from 'react';
+import { GoodsList } from './components/GoodsList';
+
+import { getAll, get5First, getRed } from './api/goods';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  render() {
+    return (
+      <div className="App">
+        <h1>Dynamic list of Goods</h1>
+        <button
+          type="button"
+          onClick={() => {
+            getAll()
+              .then((goods) => {
+                this.setState({
+                  goods,
+                });
+              });
```

Comment:
it's better to create separate functions than putting everything inside event handler in jsx